### ISSUE TITLE
Cannot hide the podman compose warning

### ISSUE THREAD

**Issue by RobusTetus** [containers/podman-compose#1033]
**Describe the bug**
Using `PODMAN_COMPOSE_WARNING_LOGS` environment variable does not hide the warning about executing external provider.

**To Reproduce**
1. Set `PODMAN_COMPOSE_WARNING_LOGS=false` either in your bashrc or directly using env command when running compose
2. Running e.g. `podman compose down` does not hide the warning as [stated in the manual](https://docs.podman.io/en/latest/markdown/podman-compose.1.html)

**Expected behavior**
Warning should not be present.

**Actual behavior**
Warning is still present as if the environment variable is not set.

**Output**
Running: `env PODMAN_COMPOSE_WARNING_LOGS=false podman compose down` or having `export PODMAN_COMPOSE_WARNING_LOGS=false` in bashrc

Produces:
```
>>>> Executing external compose provider "/usr/bin/podman-compose". Please see podman-compose(1) for how to disable this message. <<<<

```

**Environment:**
 - OS: Fedora Linux 40 Silverblue Edition
 - podman version: 5.2.2
 - podman compose version: 1.2.0

**Comment by shuckc** [comment]
This message is emitted by podman's compose command, rather than the `podman-compose` code, this is the wrong repository to report the issue in - it should be github.com/containers/podman

Nevertheless the feature was introduced July 30th https://github.com/containers/podman/pull/23442
You can see that the last tagged release of podman was 5.2.2, however this PR was added after that (see https://github.com/containers/podman/compare/v5.2.2...main ) so it is not yet released.

**Comment by RobusTetus** [comment]
> This message is emitted by podman's compose command, rather than the `podman-compose` code, so this is the wrong repository to report the issue in - it should be https://github.com/containers/podman
> 
> Nevertheless the feature was introduced July 30th [containers/podman#23442](https://github.com/containers/podman/pull/23442) You can see that the last tagged release of podman was 5.2.2, however this PR was merged after that (see [containers/podman@v5.2.2...main](https://github.com/containers/podman/compare/v5.2.2...main) ) so it is not yet released.

My bad, didn't know about that. Thank you for the explanation.
